### PR TITLE
Keyboard goes down on opening nav drawer

### LINKED ISSUE #1549: Enhancement :The keypad should disappear when opening another menu
**Summary:** 

The keypad should disappear when the user opens another menu on the screen

**Steps to reproduce:** 

1. Open the app and tap on the text field that appears
2. Click on the menu option at the top-left corner
3. The keyboard doesn't disappear.

**Expected behaviour:** 

The keyboard should disppear as soon as the menu is oppened

**Observed behaviour:** 

The keyboard doesn't disappear

**Device and Android version:** 

Pixel 2 
Android 9

GIF-200828_103705

### PR DESCRIPTION
**Pull request by sachin-0102**
Fixes #1549 

- [✓] Apply the `MifosStyle.xml` style template to your code in Android Studio.

- [✓] Run the unit tests with `./gradlew check` to make sure you didn't break anything

- [✓] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: mifosng-android/src/main/java/com/mifos/mifosxdroid/online/DashboardActivity.java [hunk 1]
```diff
@@ -180,6 +183,9 @@ public void onDrawerClosed(View drawerView) {
             @Override
             public void onDrawerOpened(View drawerView) {
                 super.onDrawerOpened(drawerView);
+                Context context = getApplicationContext();
+                InputMethodManager inputMethodManager = (InputMethodManager) context.getSystemService(Activity.INPUT_METHOD_SERVICE);
+                inputMethodManager.hideSoftInputFromWindow(drawerView.getWindowToken() , 0);
```
Comment (jawidMuhammadi) [inline mifosng-android/src/main/java/com/mifos/mifosxdroid/online/DashboardActivity.java]: We already have defined a method called `hideKeyboard` in MifosBaseActivity, use that method instead.

**Review by jawidMuhammadi — CHANGES_REQUESTED** [review]

**Review by sachin-0102 — COMMENTED** [review]
I thought it won't make a difference @jawidMuhammadi but it doesn't work that way on my device

**Comment by robustTechie** [comment]
@sachin-0102 this issue is already fixed in #1521. So no need of sending a dublicate PR.

**Review by robustTechie — COMMENTED** [review]
@sachin-0102 this issue is already fixed in #1521. So no need of sending a duplicate PR :)

**Comment by sachin-0102** [comment]
Yes sir, I saw that

**Pull request closed without merging**